index.html

```diff
@@ -39,6 +42,119 @@ <h1 id="header">Ayesha Arshad</h1>
+            <button type="button">See this project <i class="fa fa-arrow-right"></i></button>
+        </div>
+        <div class="project">
+            <img src="images/project2.png" alt="Second Project">
+            <h3>Project name goes here</h3>
+            <div class="tech-stack">
+                <span>HTML/CSS</span>
+                <span>Ruby on Rails</span>
+                <span>JavaScript</span>
+            </div>
+            <button type="button">See this project<i class="fa fa-arrow-right"></i></button>
+        </div>
+        <div class="project">
+            <img src="images/project3.png" alt="Third Project">
+            <h3>Project name goes here</h3>
+            <div class="tech-stack">
+                <span>HTML/CSS</span>
+                <span>Ruby on Rails</span>
+                <span>JavaScript</span>
+                <span>Ruby</span>
+                <span>Python</span>
+            </div>
+            <button type="button">See this project<i class="fa fa-arrow-right"></i></button>
+        </div>
+        <div class="project">
+            <img src="images/project4.png" alt="Fourth Project">
+            <h3>Project name goes here</h3>
+            <div class="tech-stack">
+                <span>HTML/CSS</span>
+                <span>Ruby on Rails</span>
+                <span>JavaScript</span>
+                <span>PHP</span>
+                <span>Node.js</span>
+            </div>
+            <button type="button">See this project<i class="fa fa-arrow-right"></i></button>
+        </div>
+        <div class="project">
+            <img src="images/project5.png" alt="Fifth Project">
+            <h3>Project name goes here</h3>
+            <div class="tech-stack">
+                <span>HTML/CSS</span>
+                <span>Ruby on Rails</span>
+                <span>JavaScript</span>
+                <span>React.js</span>
+
+            </div>
+            <button type="button">See this project<i class="fa fa-arrow-right"></i></button>
+        </div>
+        <div class="project">
+            <img src="images/project6.png" alt="Sixth Project">
+            <h3>Project name goes here</h3>
+            <div class="tech-stack">
+                <span>HTML/CSS</span>
+                <span>Ruby on Rails</span>
+                <span>JavaScript</span>
+                <span>Ruby</span>
+                <span>Node.js</span>
+            </div>
+            <button type="button">See this project<i class="fa fa-arrow-right"></i></button>
+        </div>
+    </section>
+    <section class="about-us">
+        <h1>About</h1>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit,
+            sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+            exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+        <div class="button"><small>Get my CV </small>
+            <div><i class="fas fa-arrow-down"></i></div>
+        </div>
```

Comment:
- [ ] The button (e.g., "Get my CV") should be a link `a` or `button` tag and follows the button styling: Active, Hover and Pressed. I suggest you have that fixed.

![image](https://user-images.githubusercontent.com/29071984/176938762-599f0a62-030f-4018-969a-b78bb551fd29.png)